packages/webapp/public/OneSignalSDKWorker.js

```diff
@@ -0,0 +1 @@
+importScripts('https://cdn.onesignal.com/sdks/OneSignalSDKWorker.js');
```

Comment:
We also need the updater right? At least this is what the docs mention

Reply:
For new implementations, like us, we shouldn't need it anymore. Also tested locally if the notification still works, and thankfully it still does.
![image](https://user-images.githubusercontent.com/13744167/208618500-cf9a2cf8-8e6a-4321-8332-2348f87b93a4.png)